crates/swc_css_minifier/src/compressor/media.rs

```diff
@@ -28,6 +28,22 @@ impl Compressor {
         )
     }
 
+    pub(super) fn compress_media_query_list(&mut self, media_query_list: &mut MediaQueryList) {
+        let mut already_seen: Vec<MediaQuery> = Vec::with_capacity(media_query_list.queries.len());
```

Comment:
Please use `dedup` function instead

Reply:
I mean https://github.com/swc-project/swc/blob/c4a7f21a71f619106e3293c8b5121842a13f49c9/crates/swc_css_minifier/src/compressor/selector.rs#L311-L342
Please move `deup` to `util` and use it